casbin/internal_api.cpp

```diff
@@ -147,25 +147,33 @@ bool Enforcer :: removeFilteredPolicy(const std::string& sec, const std::string&
     if (sec == "g")
         this->BuildIncrementalRoleLinks(policy_remove, p_type, effects);
 
-    if (this->adapter && this->auto_save) {
+    if (m_adapter && m_auto_save) {
         try {
-            this->adapter->RemoveFilteredPolicy(sec, p_type, field_index, field_values); \
+            m_adapter->RemoveFilteredPolicy(sec, p_type, field_index, field_values); \
         }
         catch (UnsupportedOperationException e) {
         }
     }
 
-    if (this->watcher && this->auto_notify_watcher) {
-        if (IsInstanceOf<WatcherEx>(this->watcher.get())) {
-            std::dynamic_pointer_cast<WatcherEx>(this->watcher)->UpdateForRemoveFilteredPolicy(field_index, field_values);
+    if (m_watcher && m_auto_notify_watcher) {
+        if (IsInstanceOf<WatcherEx>(m_watcher.get())) {
+            std::dynamic_pointer_cast<WatcherEx>(m_watcher)->UpdateForRemoveFilteredPolicy(field_index, field_values);
         }
         else
-            this->watcher->Update();
+            m_watcher->Update();
     }
 
     return rule_removed;
 }
 
+bool Enforcer :: updatePolicy(const std::string& sec, const std::string& p_type, const std::vector<std::string>& oldRule, const std::vector<std::string>& newRule) {
+    return true;
```

Comment:
It's not implemented yet, Are you preparing to do in the next pull request?

Reply:
Yes, I left it this way to reduce clutter in the PR. I'll be implementing it along with unit tests in the next PR.

Reply:
OK